api/src/main/java/org/apache/apex/api/ApexPlugin.java

```diff
@@ -18,12 +18,15 @@
  */
 package org.apache.apex.api;
 
+import org.apache.hadoop.classification.InterfaceStability;
+
 import com.datatorrent.api.Component;
 
 /**
  * Marker interface for ApexPlugins.
  * @param <T>
  */
+@InterfaceStability.Evolving
 public interface ApexPlugin<T extends ApexPluginContext> extends Component<T>
```

Comment:
Why ApexPlugin? It is already in org.apache.*apex*. Do we expect any other Plugins? Can plugin be in a separate package?